tools/benchmark/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # @fluid-tools/benchmark
 
+## 0.52.0
+
+Provide @benchmarkCustom feature
```

Comment:
We should provide more detail here about what the feature is and how to use it. One or two examples would be useful as well.

Reply:
Sorry @Josmithr 😄 . This change should not have been included in the PR. I merged the upstream main and now the PR only targets changes on `packages/dds/tree/src/test/shared-tree/summary.bench.ts`.